docs/design/proposals/encryption-with-vault-tokens.md

```diff
@@ -180,8 +180,53 @@ data:
   vaultBackendPath: "secret/ceph-csi-encryption/"
   vaultTLSServerName: "vault.infosec.example.org"
   vaultCAFromSecret: "vault-infosec-ca"
+  vaultClientCertFromSecret: "vault-client-cert"
+  vaultClientCertKeyFromSecret: "vault-client-cert-key"
```

Comment:
This is reasonable, in case the global configuration is done the secrets would be located in the namespace where ceph-csi is deployed. This probably should be documented (better).

Note that `vaultCAFromSecret` comes from the provisioner secret for the KMS-type `vault`. That can probably be changed as the option is not documented anywhere (so users would not be aware of it).